strings/names/ai.txt

```diff
@@ -11,20 +11,17 @@ AMEE
 AmigoBot
 Android
 Aniel
-Asimov
```

Comment:
This one is included IC and would make sense for an AI. Heck, his lawset is present